controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/pricing/PricingApiHandler.java

```diff
@@ -48,11 +49,13 @@ public class PricingApiHandler extends ThreadedHttpRequestHandler {
     private static final Logger log = Logger.getLogger(PricingApiHandler.class.getName());
 
     private final Controller controller;
+    private final PricingController pricingController;
 
     @Inject
-    public PricingApiHandler(Context parentCtx, Controller controller) {
+    public PricingApiHandler(Context parentCtx, Controller controller, PricingController pricingController) {
```

Comment:
It's also possible to move the handler itself. That way the `PricingController` interface won't be needed.

Reply:
Yeah, thanks, I'll look into that